dev_guide/builds/advanced_build_operations.adoc

```diff
@@ -122,3 +122,123 @@ endif::[]
 If the specified `NodeSelector` cannot be matched to a node with those labels,
 the build still stay in the `Pending` state indefinitely.
 ====
+
+[[dev-guide-chaining-builds]]
+== Chaining Builds
+
+For compiled languages (Go, C, C++, Java, etc.) the dependencies necessary for compilation might increase the size of the
```

Comment:
s/the dependencies necessary for compilation/including the dependencies necessary for compilation in the application image/